build-macos/CalChart.xcodeproj/project.pbxproj

```diff
@@ -117,7 +117,7 @@
 		C7AC49B014CCD51F00506C76 /* animation_frame.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7AC49AE14CCD51F00506C76 /* animation_frame.cpp */; };
 		C7AFA990117A6DBA00CC53E2 /* basic_ui.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7AFA96E117A6DBA00CC53E2 /* basic_ui.cpp */; };
 		C7AFA992117A6DBA00CC53E2 /* confgr.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7AFA973117A6DBA00CC53E2 /* confgr.cpp */; };
-		C7AFA994117A6DBA00CC53E2 /* cont_ui.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7AFA977117A6DBA00CC53E2 /* cont_ui.cpp */; };
+		C7AFA994117A6DBA00CC53E2 /* ContinuityBrowser.cpp in Sources */ = {isa = PBXBuildFile; fileRef = C7AFA977117A6DBA00CC53E2 /* ContinuityBrowser.cpp */; };
```

Comment:
We'll need similar changes in the Windows Visual Studio project, right?

Reply:
yep.  I usually do that when I'm about to make the new version available on Windows, so I only have to boot up the VM once.

Reply:
Sounds good. I have a bootable external drive with Windows, and it's also painful to use. I opened #281 to keep track of the changes that we'll ultimately need to make on Windows before the next release (also just to have an official note somewhere saying that the project will not build on Windows right now).